Sources/web3swift/Transaction/EthereumTransaction.swift

```diff
@@ -125,11 +125,12 @@ public struct EthereumTransaction: CustomStringConvertible {
         } else if self.v >= 27 && self.v <= 30 {
             d = BigUInt(27)
         }
-        if (self.chainID != nil && self.chainID != BigUInt(0)) {
+        if (self.chainID != nil && self.chainID != BigUInt(0) && self.v >= (d + self.chainID! + self.chainID!)) {
```

Comment:
we should not be using fore unwrap

Reply:
ok, while it is safe, as the first part of the if clause guarantees it is not nil by that stage of the evaluation. It is a cleaner presentation, and the original code is already a force unwrap within the conditional block. I will refactor it to remove the forced unwrap.